rkt.spec

```diff
@@ -1,31 +1,40 @@
 %global __os_install_post %{_rpmconfigdir}/brp-compress
 
-%global debug_package %{nil}
-%global provider github
-%global provider_tld com
-%global project coreos
-%global repo rkt
+%global debug_package   %{nil}
+%global provider        github
+%global provider_tld    com
+%global project         coreos
+%global repo            rkt
 
 %global import_path %{provider}.%{provider_tld}/%{project}/%{repo}
-%global commit 6dae5d5858a5364013e2504822b450ca3e307400
+#%global commit `git log | head -1 | cut -d' ' -f2
+%global commit 45cade58135d915ca1525f268369f9bef4d4fed6
 %global shortcommit %(c=%{commit}; echo ${c:0:7})
-
-Name: %{repo}
-Version: 0.7.0
-Release: 3.git%{shortcommit}%{?dist}
-Summary: CLI for running app containers
-License: ASL 2.0
-URL: https://%{import_path}
-ExclusiveArch: x86_64
-Source0: https://%{import_path}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
-Source1: README.adoc
+%global builddir %(grep AC_INIT configure.ac  | cut -d' ' -f2| tr -d \]\[,)
+
+%global version %(grep AC_INIT configure.ac  | cut -d' ' -f2| tr -d \]\[, | tr - _)
+
+# valid values: coreos usr-from-src usr-from-host
+%global stage1_usr_from host
+
+Name:       %{repo}
+Version:    %{version}
+Release:    %{shortcommit}%{?dist}
```

Comment:
define Release tag similar to original ... just re-initialize the leading counter to '1' when you have a version bump